--- a/quantum/plugins/openvswitch/ovs_quantum_plugin.py
+++ b/quantum/plugins/openvswitch/ovs_quantum_plugin.py
@@ -32,7 +32,7 @@
 import ovs_db
 
 CONF_FILE = find_config_file(
-  {"config_file": "etc/quantum/plugins/openvswitch/ovs_quantum_plugin.ini"},
+  {"config_file": "/etc/quantum/plugins/openvswitch/ovs_quantum_plugin.ini"},
   None, "ovs_quantum_plugin.ini")
 
 LOG.basicConfig(level=LOG.WARN)
